feat(backend): show wallet balance

alex1307 opened the pull request:
Description

We should show available and full wallet balance.
Wallet gRpc server alreay exposes such data [get_balance] and we will forward to the Launchpad front-end.

Motivation and Context
Full and available balance should be visible to users.

How Has This Been Tested?
The chagne is tested manually.

alex1307 committed:
feat(backend): show wallet ballance
Description

We should show available and full wallet balance.
Wallet gRpc server alreay exposes such data [get_balance] and we will forward to the Launchpad front-end.

Motivation and Context
Full and available balance should be visible to users.

How Has This Been Tested?
Maunally tested.

tarnas14 commented on applications/launchpad/backend/src/api/wallet_api.rs:
```diff
@@ -0,0 +1,95 @@
+    grpc::{GrpcWalletClient, WalletTransaction, WalletIdentity, WalletBalance},
+};
+
+
```
too much whitespace

CjS77 replied:
`cargo fmt --all` will sort this out

CjS77 commented on applications/launchpad/backend/src/main.rs:
```diff
@@ -142,6 +141,28 @@ async fn subscribe() {
     };
 }
 
+async fn check_identity() {
+    info!("Try reading wallet identity....");
+    loop {
+        sleep(Duration::from_secs(5));
+        let status = status(ImageType::Wallet).await;
+        if "running" == status.to_lowercase() {
+            let mut wallet_client = GrpcWalletClient::new();
+            match wallet_client.identity().await.map_err(|e| e.to_string()){
+                Ok(id) => {
+                    info!("Identity: {:?}", WalletIdentity::from(id));
+                    break
+                },
+                Err(err) => error!("Failed to read identity: {}", err),
+            };
+            
+        } else {
+            error!("Wallet is down");
+            continue
+        }
+    }
+}
+
```
Is this function written as intended? An infinite loop?

tarnas14 commented:
#258

alex1307 committed:
feat(backend): show wallet balance
Description

Fixing code quality issues:
Format all files and remove the developer testing code.

Motivation and Context
Clean code quality requirement.

How Has This Been Tested?
The change is tested manually.

alex1307 committed:
Merge branch 'launchpad_such_wow' into 258_wallet_balance

CjS77 reviewed (approved):
Looks like there could be some clippy issues (unused imports). Otherwise lgtm
